Casks/t/thebrowsercompany-dia@beta.rb

```diff
@@ -0,0 +1,28 @@
+cask "thebrowsercompany-dia@beta" do
```

Comment:
```suggestion
cask "thebrowsercompany-dia" do
```

We only add variants for existing casks. If a beta version is the most stable release, then we can transition to stable releases once they become available.